Dockerfile.dev

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.21-alpine3.18
```

Comment:
Is this one still needed?
I can't remember how that conversation ended.

Reply:
i think it still is for the dev docker-compose to not build the UI in the Go binary